app/views/support_interface/mailer_previews/index.html.erb

```diff
@@ -1,8 +1,8 @@
 <% @previews.each do |preview| %>
-<h3><%= preview.preview_name.titleize %></h3>
-<ul>
+<h3 class="govuk-heading-m"><%= preview.preview_name.titleize %></h3>
+<ul class="govuk-list govuk-list--bullet">
 <% preview.emails.each do |email| %>
-<li><%= link_to email, url_for(controller: 'rails/mailers', action: 'preview', path: "#{preview.preview_name}/#{email}") %></li>
+<li><%= govuk_link_to email, url_for(controller: 'rails/mailers', action: 'preview', path: "#{preview.preview_name}/#{email}") %></li>
```

Comment:
Would be nice to apply these styles to the component previews as well - obvs no need to be part of this PR, but would be cool to have consistency of styles

Reply:
I agree, can be done separately!